dashboard/settings.py

```diff
@@ -273,7 +277,6 @@ def apply_env_overrides(env: Dict[str, Any], environ: os._Environ) -> Dict[str,
 
 USE_I18N = True
 
-USE_L10N = True
 
```

Comment:
The default is True, so no need for this. This was part of Django 3.X changes. Somehow we missed it. I used https://github.com/adamchainz/django-upgrade to auto upgrade the changes to new version. Very handy

Reply:
It would be good to add a command about this library for future reference.

Reply:
Done. https://github.com/tl-its-umich-edu/my-learning-analytics/pull/1554/commits/ca01926cd8de9537ed3c7edf31d2288e20e641f6